projects/netgrif-components/schematics/view/create-view-prompt/views/public-resolver-view/files/common/__className@dasherize__.component.ts.template

```diff
@@ -0,0 +1,46 @@
+import {Component, OnDestroy, OnInit} from '@angular/core';
+import {
+    UserService,
+    SessionService,
+    AuthenticationService,
+    PublicUrlResolverService,
+    RedirectService
+} from '@netgrif/components-core';
+import {Router} from '@angular/router';
+import {Subscription} from 'rxjs';
+
+@Component({
+    selector: '<%= prefix %>-<%= dasherize(className) %>',
+    templateUrl: './<%= dasherize(className) %>.component.html',
+    styleUrls: ['./<%= dasherize(className) %>.component.scss']
+})
+export class <%= classify(className) %>Component implements OnInit, OnDestroy {
+
+    private _userSub: Subscription;
+
+    constructor(protected _userService: UserService, protected _sessionService: SessionService, protected _router: Router,
+                protected _auth: AuthenticationService, protected _publicResolver: PublicUrlResolverService,
+                protected redirectService: RedirectService) {
```

Comment:
if the redirect service is `protected` it should be prefixed with an underscore

Reply:
done